products/angular.md

```diff
@@ -115,6 +115,6 @@ months, with 1-3 minor releases for every major release. There is an
 available.
 
 Commercial support is available for all deprecated versions of Angular through the 
```

Comment:
According to https://www.herodevs.com/support/nes-angular, almost all of the releases from 6 to 17 are eligible to HeroDevs Support. Why not marking 15 - 17 versions with `extendedSupport = true`.

Reply:
@leonibr I think this is reasonable as that support is technically offered, just not needed until after EOL for most users. I think that change is good to make.